src/main/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/licences/subjectaccessrequests/SubjectAccessRequestService.kt

```diff
@@ -18,8 +19,8 @@ class SubjectAccessRequestService(
     } else {
       SarContent(
         content = Content(
-          licences = licences,
-          licenceVersions = licenceVersions,
+          licences = licences.map { it.toSAR() },
```

Comment:
rather than exposing our internal wrapper models directly to SAR - now introduce a layer of abstraction to allow us to tweak the representation more easily